presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -76,6 +77,12 @@
 public class TestLogicalPlanner
         extends BasePlanTest
 {
+    TestLogicalPlanner()
+    {
+        // in order to test testUncorrelatedSubqueries with Dynamic Filtering, enable it
+        super(ImmutableMap.of(DYNAMIC_PARTITION_PRUNING, "true"));
+    }
+
     @Test
     public void testDistinctLimitOverInequalityJoin()
```

Comment:
Please add tests like that to check that `DynamicFilter` related things are set correctly for supported queries. They should be added in class like `TestDynamicFilter` in this package.

Reply:
Done.